src/jack_client.cpp

```diff
@@ -226,7 +226,7 @@ void jack_client::calculate_plugin_order(std::vector<int> &indices)
                 map<string, int>::const_iterator p = port_to_plugin.find((*k) + cnlen + 1);
                 if (p != port_to_plugin.end())
                 {
-                    run_before.insert(make_pair<int, int>(p->second, i));
+                    run_before.insert(make_pair<const int&, unsigned int&>(p->second, i));
```

Comment:
I think it would be simpler to call the `std::pair` constructor directly here, like this:
```cpp
run_before.insert(pair<int, int>(p->second, i));
```